feat!: remove nunjucks support and `ag` CLI (#1756)
asyncapi/generator: apps/generator/docs/generator-template.md

4. Dynamically get the class name **TemperatureServiceClient** from the AsyncAPI document from the **info** object using the Parser API using the code: `asyncapi.info().title()` . It will return `Temperature Service`, then remove the spaces and add `Client` as a suffix.
5. There is no templating needed in the `__init__` function, there is only hardcoded information.

> > If you're on the fence about which templating engine you should use in your template, check out the [React render engine](https://www.asyncapi.com/docs/tools/generator/react-render-engine) documentation.

Reviewer: ```suggestion
> Only the [React render engine](https://www.asyncapi.com/docs/tools/generator/react-render-engine) is supported. Nunjucks has been deprecated.
```
